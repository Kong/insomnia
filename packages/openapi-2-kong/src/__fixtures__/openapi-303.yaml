openapi: 3.0.3
info:
  version: 1.0.0
  title: Greeting API
  termsOfService: 'https://example.com/terms/'
  contact:
    email: contact@example.com
    url: 'http://example.com/contact'
  license:
    name: Apache 2.0
    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
servers: 
  - url: http://example.org
paths: {}