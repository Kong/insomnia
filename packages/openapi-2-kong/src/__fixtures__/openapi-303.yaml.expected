{
   "_format_version": "1.1",
   "services": [
      {
         "name": "Greeting_API",
         "url": "http://example.org",
         "plugins": [],
         "routes": [],
         "tags": [
            "OAS3_import",
            "OAS3file_openapi-303.yaml"
         ]
      }
   ],
   "upstreams": [
      {
         "name": "Greeting_API",
         "targets": [
            {
               "target": "example.org:80"
            }
         ],
         "tags": [
            "OAS3_import",
            "OAS3file_openapi-303.yaml"
         ]
      }
   ]
}